firmware/console/ethernet_console.cpp

```diff
@@ -81,23 +81,6 @@ struct EthernetThread : public TunerstudioThread {
 static EthernetThread ethernetConsole;
 
 void startEthernetConsole() {
-#ifndef STM32H7
-	// TODO: why does this break H7? I thought the pins were the same?
```

Comment:
Does this remove init from nucleo nucleo-F7 config?

Reply:
In fact, now they just use the same file.